XenModel/FriendlyNames.Designer.cs

```diff
@@ -3834,7 +3834,7 @@ public class FriendlyNames {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Server &apos;{0}&apos; is now the master of pool &apos;{1}&apos;..
+        ///   Looks up a localized string similar to Server &apos;{0}&apos; is now the coordinator of pool &apos;{1}&apos;..
         /// </summary>
         public static string Message_body_pool_master_transition {

```

Comment:
master

Reply:
Will be resolving this comment and explaining on other related one for `FriendlyNames.cs`